dxr/filters.py

```diff
@@ -43,8 +43,10 @@ class Filter(object):
     """
     domain = LINE
     description = u''
+    is_reference = False
+    is_identifier = False
 
-    def __init__(self, term):
+    def __init__(self, term, enabled_plugins):
```

Comment:
We should add to the docstring an `:arg enabled_plugins` line that outlines the intended use of the plugin list by filters. (And we might as well document `term` as well, so it doesn't feel left out.)